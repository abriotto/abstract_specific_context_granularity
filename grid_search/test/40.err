  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:01, 47.77it/s] 12%|######################6                                                                                                                                                                    | 12/99 [00:00<00:01, 51.40it/s] 20%|#####################################7                                                                                                                                                     | 20/99 [00:00<00:01, 60.82it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:00<00:01, 61.86it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:00<00:00, 66.20it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:00<00:00, 60.19it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:00<00:00, 58.79it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:00<00:00, 62.95it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:01<00:00, 65.82it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:01<00:00, 71.70it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:01<00:00, 67.64it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:01<00:00, 61.55it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:01<00:00, 62.11it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 62.58it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:00, 47.71it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:00<00:00, 57.14it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:00<00:00, 49.92it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 56.81it/s]
