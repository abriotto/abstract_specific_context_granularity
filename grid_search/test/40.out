# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.99", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6967105269432068, "acc": 0.5043885707855225, "length": 3.578625440597534, "mode": "train", "epoch": 1}
{"loss": 0.6930454969406128, "acc": 0.5084270238876343, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6959680914878845, "acc": 0.5161930918693542, "length": 3.396883964538574, "mode": "train", "epoch": 2}
{"loss": 0.6941060423851013, "acc": 0.5131086707115173, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6927775740623474, "acc": 0.5080614686012268, "length": 2.6939055919647217, "mode": "train", "epoch": 3}
{"loss": 0.6926642656326294, "acc": 0.5159176588058472, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6888720393180847, "acc": 0.5365660190582275, "length": 1.95327889919281, "mode": "train", "epoch": 4}
{"loss": 0.6932696104049683, "acc": 0.49906373023986816, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6796286106109619, "acc": 0.5815439224243164, "length": 2.3306477069854736, "mode": "train", "epoch": 5}
{"loss": 0.6652207374572754, "acc": 0.6464507579803467, "length": 2.16292142868042, "mode": "test", "epoch": 5}
{"loss": 0.6627422571182251, "acc": 0.6302987337112427, "length": 2.549970865249634, "mode": "train", "epoch": 6}
{"loss": 0.6388745307922363, "acc": 0.679878830909729, "length": 2.3820223808288574, "mode": "test", "epoch": 6}
{"loss": 0.6214925646781921, "acc": 0.6881859302520752, "length": 2.7739055156707764, "mode": "train", "epoch": 7}
{"loss": 0.5941624641418457, "acc": 0.6959955096244812, "length": 2.966292142868042, "mode": "test", "epoch": 7}
{"loss": 0.5844305157661438, "acc": 0.7090651988983154, "length": 2.9587295055389404, "mode": "train", "epoch": 8}
{"loss": 0.5501719117164612, "acc": 0.7257058024406433, "length": 3.1573033332824707, "mode": "test", "epoch": 8}
{"loss": 0.543431282043457, "acc": 0.7354565262794495, "length": 3.1870834827423096, "mode": "train", "epoch": 9}
{"loss": 0.5521179437637329, "acc": 0.7387886643409729, "length": 3.4269661903381348, "mode": "test", "epoch": 9}
{"loss": 0.5117843747138977, "acc": 0.7515047788619995, "length": 3.3911890983581543, "mode": "train", "epoch": 10}
{"loss": 0.5306194424629211, "acc": 0.7523010969161987, "length": 3.4550561904907227, "mode": "test", "epoch": 10}
{"loss": 0.49888795614242554, "acc": 0.7738367915153503, "length": 3.381518840789795, "mode": "train", "epoch": 11}
{"loss": 0.5335266590118408, "acc": 0.7451866865158081, "length": 3.5505619049072266, "mode": "test", "epoch": 11}
{"loss": 0.49461475014686584, "acc": 0.7785390615463257, "length": 3.583937406539917, "mode": "train", "epoch": 12}
{"loss": 0.5217354893684387, "acc": 0.7635622620582581, "length": 3.7528090476989746, "mode": "test", "epoch": 12}
{"loss": 0.47668948769569397, "acc": 0.7896789908409119, "length": 3.579519271850586, "mode": "train", "epoch": 13}
{"loss": 0.5228914022445679, "acc": 0.7635364532470703, "length": 3.8089888095855713, "mode": "test", "epoch": 13}
{"loss": 0.4705341160297394, "acc": 0.7878512144088745, "length": 3.698115110397339, "mode": "train", "epoch": 14}
{"loss": 0.5208193063735962, "acc": 0.7747414112091064, "length": 3.8932583332061768, "mode": "test", "epoch": 14}
{"loss": 0.45317068696022034, "acc": 0.8017033934593201, "length": 3.7873940467834473, "mode": "train", "epoch": 15}
{"loss": 0.5401410460472107, "acc": 0.7659029364585876, "length": 3.9438202381134033, "mode": "test", "epoch": 15}
{"loss": 0.44784116744995117, "acc": 0.8007068634033203, "length": 3.819827079772949, "mode": "train", "epoch": 16}
{"loss": 0.5214439630508423, "acc": 0.7596998810768127, "length": 3.9606740474700928, "mode": "test", "epoch": 16}
{"loss": 0.4476121962070465, "acc": 0.8030880093574524, "length": 3.8735222816467285, "mode": "train", "epoch": 17}
{"loss": 0.5113476514816284, "acc": 0.7687265872955322, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.4316192865371704, "acc": 0.8105351328849792, "length": 3.9159884452819824, "mode": "train", "epoch": 18}
{"loss": 0.507237434387207, "acc": 0.7808988690376282, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.433062344789505, "acc": 0.8069655299186707, "length": 3.9446256160736084, "mode": "train", "epoch": 19}
{"loss": 0.5183026790618896, "acc": 0.7696629166603088, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.4247268736362457, "acc": 0.8166317939758301, "length": 3.95525860786438, "mode": "train", "epoch": 20}
{"loss": 0.5102500915527344, "acc": 0.7902622222900391, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.42370620369911194, "acc": 0.8178946375846863, "length": 3.9742684364318848, "mode": "train", "epoch": 21}
{"loss": 0.5196673274040222, "acc": 0.7874532341957092, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.41769906878471375, "acc": 0.8229207992553711, "length": 3.9783101081848145, "mode": "train", "epoch": 22}
{"loss": 0.50531405210495, "acc": 0.8014981746673584, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.4151526987552643, "acc": 0.8276306986808777, "length": 3.979963779449463, "mode": "train", "epoch": 23}
{"loss": 0.506874680519104, "acc": 0.8052434921264648, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.39329496026039124, "acc": 0.836664080619812, "length": 3.9764721393585205, "mode": "train", "epoch": 24}
{"loss": 0.5089408755302429, "acc": 0.8043071627616882, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.40141236782073975, "acc": 0.8337965607643127, "length": 3.990288734436035, "mode": "train", "epoch": 25}
{"loss": 0.5014382004737854, "acc": 0.7930712103843689, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.39247196912765503, "acc": 0.8438436985015869, "length": 3.991234064102173, "mode": "train", "epoch": 26}
{"loss": 0.48714038729667664, "acc": 0.8061797618865967, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3748098909854889, "acc": 0.8476057052612305, "length": 3.9902288913726807, "mode": "train", "epoch": 27}
{"loss": 0.5138415694236755, "acc": 0.7930710911750793, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.369137167930603, "acc": 0.840788722038269, "length": 3.995375633239746, "mode": "train", "epoch": 28}
{"loss": 0.4850817620754242, "acc": 0.8014982342720032, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.36827602982521057, "acc": 0.8473681211471558, "length": 3.9940807819366455, "mode": "train", "epoch": 29}
{"loss": 0.49329012632369995, "acc": 0.800561785697937, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.35920947790145874, "acc": 0.8470810055732727, "length": 3.9953088760375977, "mode": "train", "epoch": 30}
{"loss": 0.46936020255088806, "acc": 0.8099250197410583, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.3524262011051178, "acc": 0.8502087593078613, "length": 3.9946353435516357, "mode": "train", "epoch": 31}
{"loss": 0.48276710510253906, "acc": 0.7996254563331604, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3439680337905884, "acc": 0.8558893203735352, "length": 3.99638295173645, "mode": "train", "epoch": 32}
{"loss": 0.45534074306488037, "acc": 0.8220974206924438, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.34182196855545044, "acc": 0.84925776720047, "length": 3.991694211959839, "mode": "train", "epoch": 33}
{"loss": 0.46273481845855713, "acc": 0.8155431747436523, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3374139964580536, "acc": 0.863782525062561, "length": 3.9972710609436035, "mode": "train", "epoch": 34}
{"loss": 0.46247801184654236, "acc": 0.8117977380752563, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3344663679599762, "acc": 0.850914716720581, "length": 3.9983937740325928, "mode": "train", "epoch": 35}
{"loss": 0.45388028025627136, "acc": 0.8183521032333374, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.31948888301849365, "acc": 0.8578323721885681, "length": 3.9984447956085205, "mode": "train", "epoch": 36}
{"loss": 0.46438834071159363, "acc": 0.8183521628379822, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.328268438577652, "acc": 0.8615330457687378, "length": 3.996978282928467, "mode": "train", "epoch": 37}
{"loss": 0.46338599920272827, "acc": 0.836142361164093, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3041277527809143, "acc": 0.8713632225990295, "length": 3.9989991188049316, "mode": "train", "epoch": 38}
{"loss": 0.4692192077636719, "acc": 0.8239701390266418, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3076114058494568, "acc": 0.8760682344436646, "length": 3.9987220764160156, "mode": "train", "epoch": 39}
{"loss": 0.45900237560272217, "acc": 0.8323970437049866, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.2969757318496704, "acc": 0.8748241662979126, "length": 3.999272584915161, "mode": "train", "epoch": 40}
{"loss": 0.47410833835601807, "acc": 0.8202247023582458, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.2941906750202179, "acc": 0.8789098858833313, "length": 3.999128818511963, "mode": "train", "epoch": 41}
{"loss": 0.4683210253715515, "acc": 0.8267792463302612, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.29550081491470337, "acc": 0.8823739290237427, "length": 3.9993486404418945, "mode": "train", "epoch": 42}
{"loss": 0.4441242814064026, "acc": 0.8249065279960632, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2821464538574219, "acc": 0.8798513412475586, "length": 3.9991559982299805, "mode": "train", "epoch": 43}
{"loss": 0.4437391459941864, "acc": 0.8370785713195801, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.2773677408695221, "acc": 0.8886659145355225, "length": 3.999567985534668, "mode": "train", "epoch": 44}
{"loss": 0.4497685432434082, "acc": 0.8333333730697632, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.2743842303752899, "acc": 0.8961178064346313, "length": 3.9964990615844727, "mode": "train", "epoch": 45}
{"loss": 0.4632607400417328, "acc": 0.8286517858505249, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.2718653082847595, "acc": 0.8930579423904419, "length": 3.9991068840026855, "mode": "train", "epoch": 46}
{"loss": 0.4390481412410736, "acc": 0.8455057144165039, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.26657089591026306, "acc": 0.8918077945709229, "length": 3.9994099140167236, "mode": "train", "epoch": 47}
{"loss": 0.4502418041229248, "acc": 0.8501873016357422, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.2633158266544342, "acc": 0.892443060874939, "length": 3.9996447563171387, "mode": "train", "epoch": 48}
{"loss": 0.4382232129573822, "acc": 0.8426966071128845, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.2686542868614197, "acc": 0.8886697888374329, "length": 3.999650716781616, "mode": "train", "epoch": 49}
{"loss": 0.47154220938682556, "acc": 0.8352060914039612, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2625338137149811, "acc": 0.8968381881713867, "length": 3.9993457794189453, "mode": "train", "epoch": 50}
{"loss": 0.4576132297515869, "acc": 0.8249064087867737, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.2651367485523224, "acc": 0.8905552625656128, "length": 3.9995195865631104, "mode": "train", "epoch": 51}
{"loss": 0.44809621572494507, "acc": 0.8323971033096313, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.2604314684867859, "acc": 0.8949612379074097, "length": 3.999746084213257, "mode": "train", "epoch": 52}
{"loss": 0.4663655459880829, "acc": 0.8305243253707886, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.25531965494155884, "acc": 0.8958997130393982, "length": 3.9995951652526855, "mode": "train", "epoch": 53}
{"loss": 0.46132132411003113, "acc": 0.8445692658424377, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.2664380967617035, "acc": 0.8971503376960754, "length": 3.9993674755096436, "mode": "train", "epoch": 54}
{"loss": 0.4607090353965759, "acc": 0.8342695832252502, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.2540728747844696, "acc": 0.8936677575111389, "length": 3.998518943786621, "mode": "train", "epoch": 55}
{"loss": 0.4826809763908386, "acc": 0.8323971033096313, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.24895153939723969, "acc": 0.8968443274497986, "length": 3.999675750732422, "mode": "train", "epoch": 56}
{"loss": 0.42498934268951416, "acc": 0.8417602777481079, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2457275688648224, "acc": 0.9018787145614624, "length": 3.999755382537842, "mode": "train", "epoch": 57}
{"loss": 0.43633660674095154, "acc": 0.8323971033096313, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.24196971952915192, "acc": 0.9087840914726257, "length": 3.9993960857391357, "mode": "train", "epoch": 58}
{"loss": 0.4379095733165741, "acc": 0.8483147025108337, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.22707322239875793, "acc": 0.9037578105926514, "length": 3.999490976333618, "mode": "train", "epoch": 59}
{"loss": 0.4277936816215515, "acc": 0.8501873016357422, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.24108979105949402, "acc": 0.9109688401222229, "length": 3.9989874362945557, "mode": "train", "epoch": 60}
{"loss": 0.4402124881744385, "acc": 0.8483145833015442, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.23741309344768524, "acc": 0.9034419059753418, "length": 3.999478578567505, "mode": "train", "epoch": 61}
{"loss": 0.4555436372756958, "acc": 0.8520598411560059, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.2233109176158905, "acc": 0.9100384712219238, "length": 3.9993653297424316, "mode": "train", "epoch": 62}
{"loss": 0.44754910469055176, "acc": 0.8342695832252502, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.2241724580526352, "acc": 0.9134443998336792, "length": 3.9982192516326904, "mode": "train", "epoch": 63}
{"loss": 0.44593533873558044, "acc": 0.8408240079879761, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.22478538751602173, "acc": 0.9113122820854187, "length": 3.999774694442749, "mode": "train", "epoch": 64}
{"loss": 0.434902161359787, "acc": 0.8398875594139099, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.2187904715538025, "acc": 0.910997748374939, "length": 3.999694585800171, "mode": "train", "epoch": 65}
{"loss": 0.43063443899154663, "acc": 0.8417603373527527, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.218518927693367, "acc": 0.9116315245628357, "length": 3.9998912811279297, "mode": "train", "epoch": 66}
{"loss": 0.42483800649642944, "acc": 0.8389513492584229, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.21014206111431122, "acc": 0.9157209992408752, "length": 3.999934673309326, "mode": "train", "epoch": 67}
{"loss": 0.4221980571746826, "acc": 0.8436330556869507, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.20531825721263885, "acc": 0.9213801622390747, "length": 3.9998977184295654, "mode": "train", "epoch": 68}
{"loss": 0.4431288242340088, "acc": 0.8389513492584229, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.20956116914749146, "acc": 0.9169776439666748, "length": 3.9999070167541504, "mode": "train", "epoch": 69}
{"loss": 0.430031418800354, "acc": 0.8436330556869507, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.2143077403306961, "acc": 0.9201233983039856, "length": 3.9999351501464844, "mode": "train", "epoch": 70}
{"loss": 0.44908595085144043, "acc": 0.8314607739448547, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.20536945760250092, "acc": 0.9216822981834412, "length": 3.999551773071289, "mode": "train", "epoch": 71}
{"loss": 0.4459860920906067, "acc": 0.8455055952072144, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.21096286177635193, "acc": 0.9188551902770996, "length": 3.999615430831909, "mode": "train", "epoch": 72}
{"loss": 0.45053088665008545, "acc": 0.8445693254470825, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.22010895609855652, "acc": 0.9184440970420837, "length": 3.99619197845459, "mode": "train", "epoch": 73}
{"loss": 0.48391100764274597, "acc": 0.836142361164093, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.2066979557275772, "acc": 0.92075115442276, "length": 3.999905586242676, "mode": "train", "epoch": 74}
{"loss": 0.46575579047203064, "acc": 0.8333333730697632, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.19855421781539917, "acc": 0.9223229289054871, "length": 3.9998812675476074, "mode": "train", "epoch": 75}
{"loss": 0.4250091016292572, "acc": 0.8436330556869507, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.19828815758228302, "acc": 0.9282997250556946, "length": 3.9999475479125977, "mode": "train", "epoch": 76}
{"loss": 0.44253814220428467, "acc": 0.8473784327507019, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.19973516464233398, "acc": 0.9245249032974243, "length": 3.9999051094055176, "mode": "train", "epoch": 77}
{"loss": 0.4280281066894531, "acc": 0.8567415475845337, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.19549477100372314, "acc": 0.9226395487785339, "length": 3.999941110610962, "mode": "train", "epoch": 78}
{"loss": 0.44184207916259766, "acc": 0.8558051586151123, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.1940893530845642, "acc": 0.9248362183570862, "length": 3.999847412109375, "mode": "train", "epoch": 79}
{"loss": 0.4404301345348358, "acc": 0.8623596429824829, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.1939818412065506, "acc": 0.9298729300498962, "length": 3.999972343444824, "mode": "train", "epoch": 80}
{"loss": 0.435150146484375, "acc": 0.8520599603652954, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.17603714764118195, "acc": 0.9301785826683044, "length": 3.999765157699585, "mode": "train", "epoch": 81}
{"loss": 0.45350679755210876, "acc": 0.8492509722709656, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.17527168989181519, "acc": 0.9320715069770813, "length": 3.9998581409454346, "mode": "train", "epoch": 82}
{"loss": 0.43825364112854004, "acc": 0.8492509722709656, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.17444279789924622, "acc": 0.9301823973655701, "length": 3.9998250007629395, "mode": "train", "epoch": 83}
{"loss": 0.4363405704498291, "acc": 0.8501873016357422, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.1738346368074417, "acc": 0.9323857426643372, "length": 3.9998931884765625, "mode": "train", "epoch": 84}
{"loss": 0.42310830950737, "acc": 0.8595506548881531, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.1810608059167862, "acc": 0.9257844090461731, "length": 3.9999618530273438, "mode": "train", "epoch": 85}
{"loss": 0.4223392903804779, "acc": 0.849250853061676, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.1787528246641159, "acc": 0.930808961391449, "length": 3.999762773513794, "mode": "train", "epoch": 86}
{"loss": 0.41420039534568787, "acc": 0.852996289730072, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.17426522076129913, "acc": 0.9317498803138733, "length": 3.9997572898864746, "mode": "train", "epoch": 87}
{"loss": 0.4441896378993988, "acc": 0.8445692658424377, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.18012210726737976, "acc": 0.9336466789245605, "length": 3.999964952468872, "mode": "train", "epoch": 88}
{"loss": 0.43157297372817993, "acc": 0.8520598411560059, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.2056203931570053, "acc": 0.922623336315155, "length": 3.9996190071105957, "mode": "train", "epoch": 89}
{"loss": 0.45583364367485046, "acc": 0.8445693254470825, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.19030894339084625, "acc": 0.923897385597229, "length": 3.9999594688415527, "mode": "train", "epoch": 90}
{"loss": 0.454152911901474, "acc": 0.849250853061676, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.18198463320732117, "acc": 0.9301871657371521, "length": 3.999969959259033, "mode": "train", "epoch": 91}
{"loss": 0.4542767405509949, "acc": 0.8539325594902039, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.17025117576122284, "acc": 0.9355324506759644, "length": 3.9998111724853516, "mode": "train", "epoch": 92}
{"loss": 0.42926472425460815, "acc": 0.8548688292503357, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.17103765904903412, "acc": 0.9380200505256653, "length": 3.9994523525238037, "mode": "train", "epoch": 93}
{"loss": 0.4375568628311157, "acc": 0.849250853061676, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.1667514592409134, "acc": 0.9380277991294861, "length": 3.999228000640869, "mode": "train", "epoch": 94}
{"loss": 0.4567182660102844, "acc": 0.8520600199699402, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.16299039125442505, "acc": 0.9389849901199341, "length": 3.9997780323028564, "mode": "train", "epoch": 95}
{"loss": 0.42945683002471924, "acc": 0.8539325594902039, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.17365340888500214, "acc": 0.931445837020874, "length": 3.999988079071045, "mode": "train", "epoch": 96}
{"loss": 0.45125746726989746, "acc": 0.8623595237731934, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.16686607897281647, "acc": 0.9354518055915833, "length": 3.9983019828796387, "mode": "train", "epoch": 97}
{"loss": 0.4535793662071228, "acc": 0.8529962301254272, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.15618272125720978, "acc": 0.9386788606643677, "length": 3.999986171722412, "mode": "train", "epoch": 98}
{"loss": 0.4767197072505951, "acc": 0.8567416667938232, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.1545722335577011, "acc": 0.9415093064308167, "length": 3.9999940395355225, "mode": "train", "epoch": 99}
{"loss": 0.46438145637512207, "acc": 0.8576779365539551, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.15593694150447845, "acc": 0.9459059238433838, "length": 3.9998834133148193, "mode": "train", "epoch": 100}
{"loss": 0.4411320388317108, "acc": 0.8595504760742188, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7682292461395264
